css/header-yellow.scss

```diff
@@ -1,50 +1,91 @@
 .header-yellow {
```

Comment:
De header is al een tijdje niet meer yellow, misschien goed om deze naam erop aan te passen?

Reply:
+1